client/v3/naming/resolver/resolver.go

```diff
@@ -33,7 +34,7 @@ type builder struct {
 func (b builder) Build(target gresolver.Target, cc gresolver.ClientConn, opts gresolver.BuildOptions) (gresolver.Resolver, error) {
 	r := &resolver{
 		c:      b.c,
-		target: target.Endpoint,
+		target: strings.TrimPrefix(target.URL.Path, "/"),
```

Comment:
side note: https://github.com/grpc/grpc-go/blob/16d3df80f029f57cff5458f1d6da6aedbc23545d/clientconn.go#L1587-L1611

Reply:
The comment says `Deprecated: use URL.Path or URL.Opaque instead.`. Should we consider the `URL.Opaque` as well? Shouldn't the logic something like below? 

```
	target := target.URL.Path
	if target == "" {
		target = target.URL.Opaque
	}
	target = strings.TrimPrefix(target, "/")
```

Reply:
Hi @ahrtr,
Done.
 I looked into your query: indeed if we declare URL with no '/' after the scheme, endpoint can be written to URL.Opaque.